src/pandas_profiling/model/pandas/describe_categorical_pandas.py

```diff
@@ -147,6 +160,10 @@ def word_summary_vc(vc: pd.Series) -> dict:
     word_counts = word_counts[word_counts.index.notnull()]
     word_counts = word_counts.groupby(level=0, sort=False).sum()
     word_counts = word_counts.sort_values(ascending=False)
+
+    # Remove stop words
+    word_counts = word_counts.loc[~word_counts.index.isin(stop_words)]
```

Comment:
Perhaps only do  this when `len(stop_words) > 0`

Reply:
> On the right track, please add a unit test for `word_summary_vc`.

Where is the best place to add this unit test ?